modin/core/execution/python/implementations/pandas_on_python/partitioning/partition.py

```diff
@@ -41,17 +41,30 @@ class PandasOnPythonDataframePartition(PandasDataframePartition):
     subclasses. There is no logic for updating in-place.
     """
 
-    def __init__(self, data, length=None, width=None, call_queue=None):
-        self.data = data
+    def __init__(self, physical_data, length=None, width=None, call_queue=None):
+        self._physical_data = physical_data
```

Comment:
```suggestion
        self._data = data
```
I think `data` should be good enough.

Reply:
@devin-petersohn proposed `physical_data` naming. Devin, what do you think about `_data`/`data`?

Reply:
`_data` name is used for private property